frictionless/schema/field.py

```diff
@@ -194,6 +196,21 @@ def value_writer(cell: Any):
 
         return value_writer
 
+    @classmethod
+    def from_descriptor(  # type: ignore
+        cls,
+        descriptor: Union[IDescriptor, str],
+        **options: Any,
+    ) -> Self:
+        if isinstance(descriptor, dict):
+            if "trueValues" in descriptor.keys():
+                assert descriptor["type"] == "boolean"
+                descriptor["trueValues"] = descriptor["trueValues"]
```

Comment:
Not sure I understand this block of code =)

BTW as Table Schema is an open spec regarding custom properties we can't disable `true/falseValues` for other fields

Reply:
Indeed, you're right. This block is now useless. I've just committed to remove it.